lib/resolver.content.js

```diff
@@ -148,16 +147,27 @@ module.exports = class PodletClientContentResolver {
                         `remote resource responded with non 200 http status code for content - code: ${response.statusCode} - resource: ${outgoing.name} - url: ${outgoing.contentUri}`,
                     );
 
-                    reject(
-                        boom.boomify(
-                            new Error(
-                                `Could not read content. Resource responded with ${response.statusCode} on ${outgoing.contentUri}`,
-                            ),
-                            {
-                                statusCode: response.statusCode,
-                            },
-                        ),
-                    );
+                    const isRedirect =
+                        response.statusCode >= 300 && response.statusCode < 400;
+
+                    const statusCode = isRedirect ? 500 : response.statusCode;
```

Comment:
Boom doesn't allow us to set error codes under 400 so we just set it to 500 but thats actually not a problem because we set our own custom status code on the error `err.statusCode` and don't use the boom status code which is located at `err.output.statusCode`

Reply:
The only problem with this is that we want to know the error code when it surface in the layout. In the error handling one would like to act different if its a 302 or 301.